desktop/components/artwork_masonry_4_column/column.jade

```diff
@@ -0,0 +1,2 @@
+for artwork in column
```

Comment:
Note that I duplicated `components/artwork_masonry` into `artwork_masonry_4_column`. This can be refactored  at a later point in time into one unit that accepts `n` number of columns, but since its used in so many places this is the cleanest way to go for now.